docs/getting-started/cloudflare-workers.md

```diff
@@ -247,6 +247,50 @@ app.put('/upload/:key', async (c, next) => {
 })
 ```
 
+If the request handler were put into a separate file, you can pass the "_type struct_" through `Context`.
+
+```ts
+import { Context } from 'hono'
+
+type Bindings = {
+  MY_BUCKET: R2Bucket
+  USERNAME: string
+  PASSWORD: string
+}
+
+export const handler = async (c: Context<{
```

Comment:
We don't recommend using `Context` for `c` for the following reasons:

https://hono.dev/docs/guides/best-practices#don-t-make-controllers-when-possible